assignments/callbacks.js

```diff
@@ -2,33 +2,78 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr[0]);
 }
 
+// let firstItem = (arr, cb) => {
+//   cb(arr[0]);
+// }
+
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
 
+// let getLength = (arr, cb) => {
+//   cb(arr.length);
+// }
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
 }
 
+// let last = (arr, cb) => {
+//   cb(arr[-1]);
+// }
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  const sum = x + y;
+  cb(sum);
 }
 
+// let sumNums = (x, y, cb) => {
+//   const sum = x + y;
+//   cb(sum);
+// }
+
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  const product = x * y;
+  cb(product);
 }
 
+// let multiplyNums = (x, y, cb) => {
+//   const product = x * y;
+//   cb(product);
+// }
+
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if (list.indexOf(item)) {
```

Comment:
Line 55:

`list.indexOf(item)` returns `-1` if `item` isn't found in `list`. Otherwise it returns the first index where `item` is found in `list`. This is a problem, because 0 is a valid index, but 0 is also falsey.

If you want to use `indexOf` to answer this, you would have to alter the condition in the `if` block.

For instance, you could do:
```
if (list.indexOf(item) === -1)
```
Then if `list` does _not_ contain `item`, `indexOf` will return `-1` and the condition will pass. Then `cb(false)` will run.